common/buildcraft/silicon/BCSiliconConfig.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2017 SpaceToad and the BuildCraft team
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
+ * distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/
+ */
+
+package buildcraft.silicon;
+
+import buildcraft.api.BCModules;
+import buildcraft.core.BCCoreConfig;
+import buildcraft.lib.config.EnumRestartRequirement;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.common.config.Configuration;
+import net.minecraftforge.common.config.Property;
+import net.minecraftforge.fml.client.event.ConfigChangedEvent.OnConfigChangedEvent;
+import net.minecraftforge.fml.common.Loader;
+import net.minecraftforge.fml.common.LoaderState;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+public class BCSiliconConfig {
+
+    public static Boolean renderLaserBeams = true;
```

Comment:
Apparently I've been doing too much JavaScript at work lately if I can't remember how to properly initialize a typed variable :stuck_out_tongue_winking_eye: